src/DataCollector/SeoCollector.php

```diff
@@ -95,12 +108,12 @@ public function isHreflang(): bool
 
     public function getExternalBrokenLinks(): array
     {
-        return $this->data['externalBrokenLinks'];
+        return $this->data['externalBrokenLinks']->getValue();
```

Comment:
Most of the time, the `Data` object created by the VarDumper component, can behave like the original data.
If it's an array, it could be iterated and can access to the properties inside.
However, when returning in a method with a typehint like this, `->getValue()` is used to return a value of the inner type instead of `Data`.